go/service/user_handler.go

```diff
@@ -97,11 +97,16 @@ func (r *userHandler) userBlocked(m libkb.MetaContext, cli gregor1.IncomingInter
 	}
 	m.Debug("Got user.blocked item: %+v", msg)
 	badUIDs := make(map[keybase1.UID]bool)
+
 	for _, r := range msg.Blocks {
 		if (r.Chat != nil && *r.Chat) || (r.Follow != nil && *r.Follow) {
 			badUIDs[r.Uid] = true
 		}
 	}
+
+	// Ignore the error if we fail to block properly
+	_ = libkb.NewTracker2Syncer(m.G(), msg.Uid, true /* reverse */).Block(m, badUIDs)
```

Comment:
Could at least debug log it.

Reply:
we already log the error in`#Block` via the `Trace` function, so maybe no need to repeat it.